cmd/prune.go

```diff
@@ -61,7 +61,7 @@ func NewPruneCmd() *cobra.Command {
 	}
 
 	pruneCmd.Flags().StringVarP(&pruneopts.workspace, "workspace", "w", "WORKSPACE", "Bazel workspace file")
-	pruneCmd.Flags().StringVarP(&pruneopts.toMacro, "to_macro", "", "", "Tells bazeldnf to write the RPMs to a macro in the given bzl file instead of the WORKSPACE file.The expected format is: macroFile%defName")
+	pruneCmd.Flags().StringVarP(&pruneopts.toMacro, "to-macro", "", "", "Tells bazeldnf to write the RPMs to a macro in the given bzl file instead of the WORKSPACE file. The expected format is: macroFile%defName")
```

Comment:
ha, this catched my eye as well. I am fine either way. Mostly because gazelle used `to_macro` as well it worked for me.

Reply:
Yeah, I assume that was the origin of the name. But gazelle generally uses `-very_long_option`, while bazeldnf so far has followed the `--very-long-option` convention. To make things more interesting, bazel itself seems to use `--very_long_option`... The beauty of diversity :)